cmd/build/v1/build.go

```diff
@@ -85,8 +87,8 @@ func (bc *OktetoBuilder) Build(ctx context.Context, options *types.BuildOptions)
 		options.File = filepath.Join(path, "Dockerfile")
 	}
 
-	if err := utils.CheckIfRegularFile(options.File); err != nil {
-		return fmt.Errorf("%s: %s", oktetoErrors.InvalidDockerfile, err.Error())
+	if exists := filesystem.FileExistsAndNotDir(options.File, afero.NewOsFs()); !exists {
+		return fmt.Errorf("%s: is not a regular file", oktetoErrors.InvalidDockerfile)
```

Comment:
Nit: Not a blocker as it was as this before but, it would make sense to include `options.File` in the error message? (maybe it is not always set and that is way we didn't include it)

Reply:
it's a good idea, I've checked and the `options.File` is always set, because if empty it defaults to path + `Dockerfile`. Changed in 21f6cd9e